src/kssh/requester.go

```diff
@@ -94,7 +95,9 @@ func (r *Requester) getAllTeams() (teams []string, err error) {
 		return teams, err
 	}
 	for _, m := range memberships {
-		teams = append(teams, m.FqName)
+		if m.Role != keybase1.TeamRole_NONE {
```

Comment:
this would be more edge-case safe if we also `!= TeamRole_RESTRICTEDBOT`

Reply:
I should probably also add this in
```
func (b *Bot) getAllTeams() (teams []string, err error) {
```
, right?

Reply:
And there's also similar code in
```
func getPrincipals(conf config.Config, sr shared.SignatureRequest) (string, error) {
``` 
in `sshutils.go`

Reply:
Good idea, lets move this line to be within both `getAllTeams` functions.